src/networks/polygon_network.ts

```diff
@@ -0,0 +1,86 @@
+import { ethers } from 'ethers';
+import {
+  InsufficientBalanceError,
+  MissingWalletError,
+  PriorDustingError,
+} from '../errors';
+import { getWallet } from '../account';
+import type { Network } from '../network';
+import { PolygonNetworkConfig } from '../network_config/network_config';
+import { posRLYTestERC20 } from '../contract';
+import { gsnLightClient } from '../gsnClient/gsnClient';
+import {
+  getClaimTx,
+  getExecuteMetatransactionTx,
+} from '../gsnClient/gsnTxHelpers';
+
+async function transfer(
```

Comment:
haha we definitely need to refactor this code so that we don't dupe 95% of it just to switch networks, but this gets the job done for now.